cmd/connections.go

```diff
@@ -391,13 +384,20 @@ func PingConnection() *cli.Command {
 				printErrorForOutput(output, errors2.Wrap(err, "failed to load or create config"))
 				return cli.Exit("", 1)
 			}
+			if environment == "" {
+				err = cm.SelectEnvironment(cm.DefaultEnvironmentName)
+				if err != nil {
+					printErrorForOutput(output, errors2.Wrap(err, "failed to select the environment"))
+					return cli.Exit("", 1)
+				}
+			} else {
+				err = cm.SelectEnvironment(environment)
+				if err != nil {
+					printErrorForOutput(output, errors2.Wrap(err, "failed to select the environment"))
+					return cli.Exit("", 1)
+				}
```

Comment:
rule of thumb: if you need an `else` there's prolly a cleaner way to do it.
```suggestion
			if environment == "" {
				environment = cm.DefaultEnvironmentName
			}	
			err = cm.SelectEnvironment(environment)
			if err != nil {
				printErrorForOutput(output, errors2.Wrap(err, "failed to select the environment"))
				return cli.Exit("", 1)
			
```